kotlinlib/androidhilt/src/mill/kotlinlib/android/hilt/AndroidHiltTransformAsm.scala

```diff
@@ -0,0 +1,109 @@
+package mill.kotlinlib.android.hilt
+
+import com.android.build.api.instrumentation.{ClassContext, ClassData}
+import dagger.hilt.android.plugin.transform.AndroidEntryPointClassVisitor
+import mill.*
```

Comment:
Is this `mill`-import used anywhere?

Reply:
no, good catch, I removed it with the rebase